PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Thanks for your contribution!
+
+Please take a quick look at those things down there. They're quite important.
+Really! We wrote them for you. Yes you! With utmost care. Read them.
```

Comment:
Hey, I know you have followed main repositories' template but it will look nicer if add a `,` between `Yes` and `you` IMO. 😃

Reply:
👍